pkg/products/amqonline/authservices.go

```diff
@@ -14,6 +16,18 @@ func GetDefaultAuthServices(ns string) []*v1beta1.AuthenticationService {
 			},
 			Spec: v1beta1.AuthenticationServiceSpec{
 				Type: "standard",
+				Standard: &v1beta1.AuthenticationServiceSpecStandard{
+					Datasource: &v1beta1.AuthenticationServiceSpecStandardDatasource{
+						Type:     v1beta1.PostgresqlDatasource,
+						Host:     "postgresql",
+						Port:     5432,
+						Database: "amqonline",
+						CredentialsSecret: corev1.SecretReference{
+							Name:      "postgresql",
+							Namespace: ns,
+						},
+					},
+				},
```

Comment:
@grdryn I'm guessing there's more changes coming to create the `Postgres` CR for CRO as well?

Reply:
Yeah, sorry, I meant to mark the PR as WIP. All should be done now.